tools/install-MXU.py

```
@@ -82,6 +82,13 @@ def install_resource():
         install_path,
     )
     
+    # 复制 bbc_team_config.json 到根目录（与 interface.json 同级）
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)